src/archivematicaCommon/lib/workflow.py

```diff
@@ -0,0 +1,66 @@
+import sys
+
+import grpc
+
+from protos import workflow_pb2
+
+
+def _rendezvous_exception(func):
+    def _decorated(*args, **kwargs):
+        try:
+            return func(*args, **kwargs)
+        except grpc._channel._Rendezvous as e:
+            if e.code() == grpc.StatusCode.UNAVAILABLE:
+                raise ServerUnavailableError(e.code())
+    return _decorated
+
+
+class Error(Exception):
+    def __init__(self, message):
+        self.message = 'Workflow client error: ' + message
+
+
+class ServerUnavailableError(Error):
+    def __init__(self, code):
+        self.code = code
+        self.message = 'server unavailable, code ' + self.code
```

Comment:
Can't concatenate a str & int - `self.message = 'server unavailable, code %s' % (self.code,)` will work.  Should probably do the same precaution above.